R/utils.R

```diff
@@ -21,11 +21,11 @@ named_commas <- function(x) {
 commas <- function(...) paste0(..., collapse = ", ")
 
 unique_table_name <- function() {
-  # Needs to use option to unique names across reloads while testing
-  i <- getOption("dbplyr_table_name", 0) + 1
-  options(dbplyr_table_name = i)
-  sprintf("dbplyr_%03i", i)
+  vals <- c(letters, LETTERS, 0:9)
```

Comment:
@mgirlich FYI I changed this so that we can generate new table names across multiple threads. I didn't change any test output but I wanted to let you know in case you were surprised in the future.